README.md

```diff
@@ -78,62 +78,30 @@ moveroplot C-1E_ch/C-2E_ch --lt_ranges 07-12,19-24,61-72 --input_dir /scratch/os
 ```
 
 ## Plotting Pipeline and Output
+### Plotting Multiple Model Versions
 
 ### Spatial Verification
+> Relevant File: [station_scores.py](src/moveroplot/station_scores.py)
```

Comment:
Add a blank line to avoid nesting the image in the quote

Reply:
Done